pkg/controller/operators/olm/operator_test.go

```diff
@@ -2358,7 +2358,7 @@ func TestSyncOperatorGroups(t *testing.T) {
 					annotatedDeployment,
 				},
 				targetNamespace: {
-					withAnnotations(targetCSV.DeepCopy(), map[string]string{"olm.targetNamespaces": targetNamespace + "," + operatorNamespace, "olm.operatorGroup": "operator-group-1", "olm.operatorNamespace": operatorNamespace}),
+					withAnnotations(targetCSV.DeepCopy(), map[string]string{"olm.operatorGroup": "operator-group-1", "olm.operatorNamespace": operatorNamespace}),
```

Comment:
Why aren't we using the constants here?

Reply:
I fixed up all of the string references now.